tests/behat/features/capabilities/notifications/notification-emails.feature

```diff
@@ -122,3 +122,18 @@ Feature: Receive email notifications and choose frequency
       | Cathy Willis mentioned you in a post                       |
       | Thomas Miller mentioned you in a post                      |
       | the notifications above are sent to you as a Weekly mail   |
+
+  @email-spool
+  Scenario: User is able to get no emails for activities if he doesn't login at lease once.
+    Given I set the configuration item "social_swiftmail.settings" with key "do_not_send_emails_new_users" to 0
+    And I set the configuration item "system.site" with key "name" to "Open Social"
+    And users:
+      | name    | mail                   | status | field_profile_first_name | field_profile_last_name |
+      | user1   | mail_user1@example.com | 1      | Christopher              | Conway                  |
+      | user2   | mail_user2@example.com | 1      | Cathy                    | Willis                  |
+    And I am logged in as "user2"
+    And I am on the homepage
+    And I fill in "Say something to the Community" with "You're not going to be notified of this [~user1]!"
+    And I press "Post"
+    And I wait for the queue to be empty
+    Then I should not have an email with subject "Notification from Open Social" and "Cathy Willis mentioned you" in the body
```

Comment:
@denysyukm It would be good to also add scenario where we have 

`Given I set the configuration item "social_swiftmail.settings" with key "do_not_send_emails_new_users" to 1`
`Then I should have an email with subject "Notification from Open Social" and "Cathy Willis mentioned you" in the body`

Can you add that?

Reply:
@ribel , Good point! I added positive scenario

Reply:
I see your test is failing now - could you check it again?